src/components/ebay-lightbox-dialog/examples/02-scrolling/template.marko

```diff
@@ -0,0 +1,28 @@
+<ebay-lightbox-dialog
+    a11y-close-text="Close Dialog"
+    aria-labelledby="dialog-title"
```

Comment:
I thought the `aria-labelledby` was taken care of internally and not an option anymore?